README.md

```diff
@@ -1,15 +1,16 @@
 # pz-rail-hub
 
+[![Read the Docs](https://img.shields.io/readthedocs/qp-flexzboost)](https://lsstdescrail.readthedocs.io/en/latest/index.html)
+[![PyPI](https://img.shields.io/pypi/v/hub?color=blue&logo=pypi&logoColor=white)](https://pypi.org/project/pz-rail-hub/)
+[![codecov](https://codecov.io/gh/LSSTDESC/pz-rail-hub/branch/main/graph/badge.svg)](https://codecov.io/gh/LSSTDESC/rail)
+
 [![Template](https://img.shields.io/badge/Template-LINCC%20Frameworks%20Python%20Project%20Template-brightgreen)](https://lincc-ppt.readthedocs.io/en/latest/)
```

Comment:
Not quite the same template.   But do you have readthedocs for the RAIL template?

Reply:
Fixed